docs/user-guide/time-and-costs/time-and-costs-faq/README.md

```diff
@@ -29,6 +29,10 @@ Unfortunately this is not possible at the moment. There's already a feature requ
 
 Yes, you can use the "My spent time" widget on My Page and use the filters there.
 
+## Can I log time in a different unit than hours, e.g. in days?
+
+No, it is not (yet) possible to log time in days or any other units than hours. However, you can use decimal places, like 0.25 hours.
```

Comment:
Maybe to make it even more clear, we could say "...any other units besides hours"?